Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+
+Every 4 hours the wallet is on-line, it looks for old coins that can
+accrue interest and moves them from 'Balance' to 'Stake' then it works
+on staking. Once staked, it moves the staked balance back to
+balance.
+
+## I lose my stakeweight after receiving PoR? The "maturecoin-creationprocess" is starting again after each mined PoR?
+
+Yes that is normal, because your node builds up a set of stakeable coins
+and once you stake them the process takes about 4 more hours to get new
+mature coins, unless some have been found in between (it's a constant
+process).
+
+## Do I have to leave my wallet unencrypted to be able to stake?
+
+Well, if you encrypt your wallet and enter `walletpassphrase
+\[passphrase\] 86400`, that would unlock it for 24 hours for staking.
+
+Your wallet file remains encrypted and since you do not have to enable
+external RPC access for mining anymore, it's safe aside from physical
+access or perhaps memory leakage.
+
+## Can I stake with encrypted wallet?
+
+Yes, but you have to unlock it first. It's okay to unlock it "for
+staking only".
+
+The console walletpassphrase command does what clicking the lock button
+does.
+
+To lock Type
+
+`walletlock`
+
+to unlock for a certain time type
+
+`walletpassphrase PASSWORD TIMEOUT`
+
+to unlock type
+
+`walletpassphrase PASSWORD`
+
+to unlock only for staking type
+
+`walletpassphrase PASSWORD TIMEOUT true`
+
+## How do you turn staking on and off?
+
+`walletpassphrase "your passphrase in double quotes" timeout true`
+
+where timeout is a number of seconds, e.g. 10000 and true is a toggle
+that unlocks the wallet for staking
+only.
+
+# BOINC
+
+## Choosing BOINC projects: In BOINC, is there still a defined whitelist of "projects" or can we use any project now that has Gridcoin as a Team Name?
+
+Running "list projects" in console will list all the whitelisted
+projects you could possibly contribute to that would help your Gridcoin
+Research
+wallet.
+
+## If it says "non participating project" does that mean that they aren't accepted for Gridcoin purposes?
+
+That means you aren't participating in those projects or the e-mail
+address you have in the conf file doesn't match the e-mail address used
+in your BOINC manager on that machine. For every project you join, you
+also have to join the Gridcoin team on that project. Any changes about
+e-mail address on a project or team membership on a project take around
+a day to
+propagate.
+
+## I use BOINC with one username but on a few machines - is it enough to have one wallet with the proper boinc\_project\_email address, or do I have to have a separate wallet on each machine?
+
+You only need one wallet for each BOINC account/CPID (Cross Project
+Identifier). You can use BAM (BOINC Account Manager) to combine more
+projects in one CPID. Here is a guide
+<http://grcnation.com/en/bam-boinc-account-manager/> . Your reward is

```

Comment:
This website isn't online anymore. Needs to be replaced with something else

Reply:
What is suitable?